requirements.txt

```diff
@@ -5,4 +5,6 @@ androguard==3.3.0
 rsa==4.0
 biplist==1.0.3
 requests==2.21.0
+yara
+bs4
```

Comment:
I did find this for `bs4`: 
https://github.com/MobSF/Mobile-Security-Framework-MobSF/blob/18105ef222f05d323dd5fa1911158719aab12b2e/scripts/check_android_permissions.py#L6
and this for `yara`:
https://github.com/MobSF/Mobile-Security-Framework-MobSF/blob/cc4edc272f92c4b449c1bc9f81065e8cd1b98cf5/StaticAnalyzer/tools/apkid/apkid.py#L42

I am removing `yara` since apkid is an additional requirement :slightly_smiling_face:

Reply:
Scripts are auxiliary and independent. Not always directly used by MobSF. But let's have bs4 in requirements